src/components/Resource/Wizard/ModelVersionWizard.tsx

```diff
@@ -228,7 +228,8 @@ export function ModelVersionWizard({ data }: Props) {
 
   const hasFiles = modelVersion && !!modelVersion.files?.length;
 
-  const postId = modelVersion?.posts?.[0]?.id;
+  // Filter to posts belonging to the owner of the model
+  const postId = modelVersion?.posts?.filter((post) => post.userId === modelData?.user.id)?.[0]?.id;
```

Comment:
we could probably do this in the backend, but would require more changes to do a raw query for it